scripts/pre_commit_linter.py

```diff
@@ -470,22 +470,19 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
-def _get_all_files():
-    """This function is used to parse the arguments entered by the user, and
-    returns the list of all files for linting and pattern checks.
+def _get_all_files(input_files, input_path):
+    """This function is used to return the files which needs to linted and
```

Comment:
linted --> be linted. (Next time please proofread your changes before submitting, otherwise this delays the review process.)

Reply:
Done